packages/edit-site/src/index.js

```diff
@@ -12,6 +12,7 @@ import { render } from '@wordpress/element';
  * Internal dependencies
  */
 import './hooks';
+import './store';
```

Comment:
I feel this config should be reused somehow as well, some options:

 - Move it to the block-editor store (doesn't make sense for me as it's just a temporary location for the component)
 - Use a store in the new package, downside is that it makes it a singleton package (another one). (option we used everywhere for the moment)

Options we didn't use anywhere yet but could be good for reusabliity

 - Export a store config from the new package and leave the responsibility to register it to the caller `wp.data.registerStore( 'core/edit-post-full-screen', fullscreenStoreConfig )`
 - Don't use a store, use a React context provider.

I don't think it needs to be solved in this PR, but something to think about. 

cc @aduth if you have thoughts.

Reply:
To clarify, by "config", do you mean the user's general preference of "full site mode" being activated or not, across different implementations of the block editor?

Reply:
yes